Mlem/Views/Shared/Cached Image.swift

```diff
@@ -108,18 +109,15 @@ struct CachedImage: View {
                                     }
                                     try data.write(to: quicklook)
                                     await MainActor.run {
-                                        bigPicMode = quicklook
+                                        quickLookPaths.url = quicklook
                                     }
                                 } catch {
                                     print(String(describing: error))
                                 }
                             }
                         }
-                        .quickLookPreview($bigPicMode)
-                        .onChange(of: bigPicMode) { mode in
-                            if mode == nil, let dismissCallback {
-                                dismissCallback()
-                            }
+                        .fullScreenCover(item: $quickLookPaths.url) { url in
+                            QuickLookView(urls: [url])
```

Comment:
This breaks the functionality of marking posts read when the preview is opened--it should be a pretty easy fix, it just needs to call `dismissCallback()` at any point after the viewer is opened.

If you go with the `URL?` suggested above, the previous `onChange` will work--otherwise you'll need to figure something out, since I think applying that `onChange` to an `@EnvironmentObject` is going to mark all rendered posts as read.

Reply:
Reverted to the old `URL?` logic, we won't need that `QuickLookPaths` object if multi-image support isn't a thing anyway.